cli/module_graph2.rs

```diff
@@ -666,6 +674,17 @@ impl Graph2 {
     Ok(result)
   }
 
+  fn resolve_specifier<'a>(
+    &'a self,
+    specifier: &'a ModuleSpecifier,
+  ) -> &'a ModuleSpecifier {
+    let mut s = specifier;
+    while let Some(redirect) = self.redirects.get(s) {
+      s = redirect;
+    }
+    s
+  }
```

Comment:
I wonder if we should guard against infinite redirect loop here - it might already be handled in `SourceFileFetcher` tho...

Reply:
It isn't specifically guarded, but it would like cause errors elsewhere first (in `file_fetcher`).  I will take a look though.

Reply:
I have guarded against infinite loops and excessive redirections... While it might be overkill, it is good to have clear logic to handle those edge cases.